lib/pure/os.nim

```diff
@@ -1407,8 +1407,9 @@ proc absolutePath*(path: string, root = getCurrentDir()): string =
 proc absolutePathInternal(path: string): string =
   absolutePath(path, getCurrentDir())
 
-proc normalizePath*(path: var string) {.rtl, extern: "nos$1", tags: [].} =
-  ## Normalize a path.
+proc normalizePath*(path: var string, exe = false) {.rtl, extern: "nos$1", tags: [].} =
+  ## Normalize a path. If `exe == true`, on posix, `./` is prepended if `path`
+  ## doesn't contain `/` and is not `"", ".", ".."`.
```

Comment:
```suggestion
  ## Normalize a path.
  ##
  ## If `exe == true`, on posix, `./` is prepended if `path`
  ## doesn't contain `/` and is not `"", ".", ".."`.
```

The layout should always be:

* Short description
* Long Description
* Detail about differences in behaviour depending on params
* Exceptions

Reply:
applied fix